### PR TITLE
A new website!

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: source/_static/style.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+}
+p.content.noheader {
+	padding-top: 15px;
+}
+li.dropdown.global-container {
+	display: none !important;
+}
+/*Navbar*/
+.navbar {
+	background-color: #212121;
```
Comment (Cadair) [inline source/_static/style.css]: I am not a massive fan of this dark blue background colour. Maybe black instead?

File: source/about.rst [hunk 2]
```diff
@@ -0,0 +1,212 @@
+:orphan:
+
+Our Mission
+===========
+
+.. raw:: html
```
Comment (Cadair) [inline source/about.rst]: This, and many other places should be replaced. Where possible we want to be using restructred text markup for all the content.
I appreciate this is a good stop gap for now, but we should put it on the list.

File: source/conf.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+# sys.path.insert(0, os.path.abspath('.'))
+# -- General configuration ------------------------------------------------
+# If your documentation needs a minimal Sphinx version, state it here.
+# needs_sphinx = '1.0'
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+import sphinx_bootstrap_theme
+extensions = ['sphinx.ext.intersphinx',
+'sphinx.ext.githubpages']
```
Comment (Cadair) [inline source/conf.py]: indentation.

File: source/index.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.. raw:: html
+
+    <embed>
+   		<div class="center" style="vertical-align:middle;">
+   			<img src="_static/img/sunpy_icon.svg" alt="SunPy Logo" width="20%">
```
Comment (Cadair) [inline source/index.rst]: I think `width=30%` looks better, but a bit bigger would be nice.

File: source/conf.py [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+# 'preamble': '',
+# Latex figure (float) alignment
+#
+# 'figure_align': 'htbp',
+}
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+(master_doc, 'SunPy.tex', u'SunPy Documentation',
```
Comment (Cadair) [inline source/conf.py]: indentation.

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+language: python
+
+python:
+  - 3.5
```
Comment (nabobalis) [inline .travis.yml]: I do not think we need 2 versions of python. 3.6 should be enough.

Comment (DuyguKeskek) [reply .travis.yml]: Done.

File: source/index.rst (hunk 4 above)
Comment (DuyguKeskek) [reply source/index.rst]: Done.

File: source/conf.py (hunk 5 above)
Comment (DuyguKeskek) [reply source/conf.py]: Done.

File: source/conf.py (hunk 3 above)
Comment (DuyguKeskek) [reply source/conf.py]: Done.

File: source/about.rst (hunk 2 above)
Comment (DuyguKeskek) [reply source/about.rst]: If time allows, I'll have a look at that.

**Comment by nabobalis** [comment]
The reason it fails is that you need to install the fix branch and not the master branch.

File: requirements.txt [hunk 7]
```diff
@@ -0,0 +1,3 @@
+sphinx
+sphinx_bootstrap_theme
+git+https://github.com/Cadair/ablog.git
```
Comment (nabobalis) [inline requirements.txt]: This should be: git+https://github.com/Cadair/ablog.git@tempfix

File: .travis.yml [hunk 8]
```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+install: "pip install -r requirements.txt"
+
+before_script: ablog build
```
Comment (Cadair) [inline .travis.yml]: do we need this? Won't `make html` run ablog?

Comment (DuyguKeskek) [reply .travis.yml]: Uh, my mistake. I thought ablog won't build the rst files without `ablog build` , but `make html` seems enough. I'll change the yml file in the next commit.

Comment (nabobalis) [reply .travis.yml]: Yeah, I think this line needs to be deleted. it seems to be why travis is failing to build.

**Comment by Cadair** [comment]
So as we are moving to the docs for a while, this is not a list of things to fix, it's a list of things I think we should discuss, this is mainly about what we want the final structuring to look like:

- Documentation: Do we want this to be a drop down for SunPy and affiliated packages?
- "Support Us" is support the right phrasing here would "get involved" not be better?
- Affiliated Projects: This *could* be under "SunPy Project" depends what we want to do with it.

Finally, I think it would be good to have (eventually) the following things under "SunPy Project":

* A description of the project
* The SunPy board, with the roles on the board split out.
* A section for roles: Lead Dev, Release Manager, Community Manager, SoC Organizer, (me, nab, jack David respectively).
* Numfocus (best here or somewhere else?)
* An automatically generated list of contributors broken down by affiliated package.

**Comment by nabobalis** [comment]
Quick response here:
1. Yes 
2. I prefer Support Us but I'm not wed to it.
3. I do think that the current bar has too many items since when we resize to a smaller window it doubles up. I wonder if Documentation should be moved under Get Help. 

The rest of the stuff would be good to add sometime. I think the website will see alot of changes after GSOC anyway as we keep adding more information and the blog gets up and running.

**Comment by Cadair** [comment]
I think documentation should remain on the top bar, also bear in mind the top bar will be built into the theme, so the idea is the docs and the site have (mostly) the same links on it.

Perhaps we should refactor the about page and turn what is currently termed "SunPy Project" to be "people" or something?

Also, we need a "Get Started" page for the link on the home page, perhaps we could make that a redesigned about page?

I think we need to sit down and think about who is coming to this site and what information they want.

**Comment by mbobra** [comment]
The website looks good!

-- I'm confused as to why the "SunPy Project" tab is separate from the content in the "About" tab -- especially since some of the wording (e.g. "SunPy is a living code base with many contributors. Anyone can and is welcome to get involved.") is the same. Also I think it should be alphabetically organized.

-- I guess the "Documentation" tab is not finished yet, which is fine. I suggested to Stuart a while ago to enable the notebooks in the Gallery on the old site with Binder so they can run on Google Container Engine. I'm not sure if this is in the scope of your project but if you could do it, I think it would be super cool! Here is an example: https://github.com/mbobra/calculating-spaceweather-keywords. You can click on the black and red "launch binder" button to run the notebooks in Binder. 

Okay that is about it.

**Comment by DuyguKeskek** [comment]
@mbobra Thanks for the review ! 
-- We will probably have to redesign the About tab. The aim was to separate the team from about tab like Astropy does, but I also feel that the SunPy board should be somehow placed in about tab since it is the first place for someone to look for the details of the project.  We can also delete the team tab on the about section and fill the SunPy Project tab with much more information.
-- I haven't started working on the documentation yet, but that's a clever idea ! If time allows, I can have a look at this later. Thanks again for all the recommendations.

**Comment by DuyguKeskek** [comment]
@Cadair I think the documentation should definitely remain on the top navbar. We can change "SunPy Project" to be only "Team" -this is what Astropy does currently- 

I agree on redesigning the about page. It is too much text and not that interesting right now. We can link the section with "Get Started" button, but we have to rearrange the content, decide on which tabs we will use, and maybe keep it really simple and short.

**Comment by ehsteve** [comment]
This is looking like a great start! The page is nice and uses responsive design which is great.

A few quick comments,

- the shortcuts on the left make a strange jump when you first start scrolling down before it begins to follow the scrolling.
- the shortcuts on the left used to show where you were with a small highlight but does not anymore.
- the shortcuts on the left, if very long, do not scroll so there is no way to get to the ones that are off the screen. This is most apparent on the 'All Posts' blog page
- the front page links "Getting Started" and "Install SunPy" have no buffer space below them. Clicking those links also does nothing.
- Does the SunPy blog still provide an RSS feed?
- Clicking on an author of a blog entry performs a search of all entries by that author which is unexpected behavior (from my perspective). Also once you go to view an article the author information is gone.
- So many Russ!

**Comment by Cadair** [comment]
Merged into https://github.com/sunpy/sunpy.github.io 🎆 🎆

**Pull request closed without merging**